os/android/drmhwctwo.cpp

```diff
@@ -443,8 +446,14 @@ HWC2::Error DrmHwcTwo::HwcDisplay::PresentDisplay(int32_t *retire_fence) {
   for (std::pair<const hwc2_layer_t, DrmHwcTwo::HwcLayer> &l : layers_) {
     switch (l.second.validated_type()) {
       case HWC2::Composition::Device:
-      case HWC2::Composition::Cursor:
         z_map.emplace(std::make_pair(l.second.z_order(), &l.second));
+        cursor_z_order = std::max(cursor_z_order, l.second.z_order());
```

Comment:
Instead of this ,  I think we can do  std::max(cursor_z_order, <last layer in map>.second.z_order()); ?
map should order the layers by z order, so last layer should have highest z order.